generator/tests/test_vogtle_data_generator.py

```diff
@@ -0,0 +1,162 @@
+"""PyTest Vogtle Data Generator File
+
+Tests all Vogtle related synthetic data generator
+
+"""
+import sys
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+from vogtle_data_generator import VogtleDataGenerator
+import vogtle_data_generator as vdg
+from common import validations
+from . import test_values as tv
+
+
+class TestVogtleDataGenerator(unittest.TestCase):
+    """ Class for all Vogtle Data Generator testing
+    """
+    VogtleDataGenerator = VogtleDataGenerator(directory='test_data/')
+
+    def test_generate_inspections(self):
+        """ Test inspection data generation
+        """
+        config = tv.VOGTLE_CONFIG['inspections']
+        self.VogtleDataGenerator.generate_inspections(config['row_count'])
+
+        assert validations.validate_csv(filename=config['filename'],
```

Comment:
The way you assigned the arguments here is good and I would not change it.

If you ever wanted to retrieve multiple items from a dictionary sometimes it's nice to use `itemgetter`.  Here your code would look like this:

```python
from operator import itemgetter

assert validations.validate_csv(
            *itemgetter('filename', 'header', 'col_count', 'row_count')(config)
        )
```